CHANGELOG.md

```diff
@@ -3,6 +3,12 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
 
+## [0.3.3] - 2019-07-08
+
+### Changed
+* [Bug 1561617](https://bugzilla.mozilla.org/show_bug.cgi?id=1561617): part 2 - FennecVersion accepts 68.Xa1.
```

Comment:
Tiny nit: not sure if "part 2" is useful in this context as nor the https://github.com/mozilla-releng/mozilla-version/pull/56/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR15 had "part 1" counterpart.

Reply:
True. I added `part 1` to avoid confusion. Thanks for the super quick review!